Source/WebGPU/WGSL/tests/lit.cfg

```diff
@@ -36,7 +36,7 @@ config.substitutions.append(('%metal-compile', (
     "function metal_compile() {"
     "     set -e -o pipefail;"
     f"    {wgslc} --dump-generated-code '%s' \"$1\" > '%t.metal' || exit 1;"
-    f"    xcrun -sdk macosx metal -Werror {' '.join(ignored_warnings)} -c '%t.metal' -o /dev/null;"
+    f"    xcrun -sdk macosx metal -D__wgslMetalAppleGPUFamily=9 -Werror {' '.join(ignored_warnings)} -c '%t.metal' -o /dev/null;"
```

Comment:
I tried to find an easy way to get the actual family from the command line, but couldn't get it. I guess it's not really relevant for wgslc anyway, right?

Reply:
that's right since wgslc doesn't create PSOs